packages/mui-envinfo/test/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/material": "^5.15.0",
```

Comment:
Hmmm, why wasn't this picked up by the latest release process? 😕 @siriwatknp keep an eye on this for the next release.

Reply:
Thanks for pointing out!